programs/cleos/main.cpp

```diff
@@ -1735,6 +1737,8 @@ int main( int argc, char** argv ) {
    getTable->add_option( "-k,--key", table_key, localized("The name of the key to index by as defined by the abi, defaults to primary key") );
    getTable->add_option( "-L,--lower", lower, localized("JSON representation of lower bound value of key, defaults to first") );
    getTable->add_option( "-U,--upper", upper, localized("JSON representation of upper bound value value of key, defaults to last") );
+   getTable->add_option( "-t,--seckeytype", sec_key_type, localized("Secondary key type(i64,i128,i256,float64,float128)"));
+   getTable->add_option( "-p,--secindexposition", sec_index_position, localized("Secondary index position, default to 0(first secondary index)"));
```

Comment:
Since we will support diff primary key types in the future, the abbreviated `-t -p` are too generic. Also too much implementation detail here. Seems odd to talk about first, second, third, etc secondary index. From the users perspective it should just be first (primary), second, third, etc.

How about:
```
"--index", "Index number, 1 - primary (first), 2 - secondary index (in order defined by mult_index), 3 - third index, etc."
"--key-type", "The key type of --index, primary currently only supports (i64), all others support (i64, i128, i256, hex256, float64, float128)"
```

Reply:
"-k,--key" should either be removed or made to work. Currently, it is not used for anything.

Reply:
@wanderingbort suggested also supporting "primary", "secondary", etc as args to --index so the user does not have to use literal numbers.